python/pip_install/create_incremental_repo/BUILD

```diff
@@ -0,0 +1,21 @@
+load("@rules_python//python:defs.bzl", "py_library", "py_test")
+load("//python/pip_install:repositories.bzl", "requirement")
+
+py_library(
+    name = "lib",
+    srcs = ["__init__.py"],
+    deps = [requirement("pip")]
+)
+
+py_test(
+    name = "test_create_incremental_repo",
```

Comment:
if you change the word "incremental" then change here too, like maybe "python/pip_install/translate_lockfile_to_bzl" or sth